### PR TITLE
Fix #26: Add liquibase to the docker image

### LINKED ISSUE #26: Add liquibase to the docker image

### PR DESCRIPTION
**Pull request by zcgandcomp**

### CODE REVIEW

**Comment by banterCZ** [comment]
Thanks for adding missing indexes and fixing the liquibase at docker. We have to decide what to do with https://github.com/wultra/user-data-store/blob/develop/docs/sql/postgresql/create-schema.sql It should be deleted or updated.

**Comment by zcgandcomp** [comment]
Regarding create-schema.sql - just tell me what is expected. My idea was we that going to generate (if needed) the script from the liquibase changesets. Or we can maintain them for now manually and resolve the redundancy later.

**Comment by banterCZ** [comment]
> Or we can maintain them for now manually and resolve the redundancy later.
Ok, we may resolve concept in another PR, but let's edit the script manually to keep it in sync.

File: docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Zdenek Cerny">
+        <comment>Create indexes on table audit_log and audit_param</comment>
```
Comment (banterCZ) [inline docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml:7]: Missing `exists precondition`. Moreover, each index should be extra changeset. See https://github.com/wultra/enrollment-server/blob/develop/docs/db/changelog/changesets/enrollment-server/1.4.x/20230314-audit.xml

**Review by banterCZ — CHANGES_REQUESTED** [review]

File: docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+    <changeSet id="`" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Lubos Racansky">
```
Comment (banterCZ) [inline docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml:5]:
```suggestion
    <changeSet id="1" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Lubos Racansky">
```

File: docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        </preConditions>
+        <comment>Create a new index on audit_log(timestamp_created)</comment>
+        <createIndex tableName="audit_log" indexName="audit_log_timestamp">
+            <column name="timestamp_created" />
+        </createIndex>
+        <!-- no rollback on purpose, the audit tables may be shared across several components -->
+        <rollback />
+    </changeSet>
+
+    <changeSet id="1" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Lubos Racansky">
```
Comment (banterCZ) [inline docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml:19]:
```suggestion
    <changeSet id="2" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Lubos Racansky">
```

**Review by banterCZ — APPROVED** [review]

**Pull request merged**